### PR TITLE
Handle upgrade of cortx-py-utils package

### PR DESCRIPTION
**Pull request by sachinpunadikar**
While upgrading the cortx-py-utils package, as part of pre-uninstall
phase, all required packages were getting removed. This is not
expected in upgrade scenarios.
Made corrections in pre-uninstall script.

### CODE REVIEW

File: py-utils/utils-pre-uninstall [hunk 1]
```diff
@@ -3,4 +3,8 @@
 PIP=pip3.6 
 POST_INSTALL_REQUIREMENTS=/opt/seagate/cortx/utils/conf/requirements.txt
 
-sudo $PIP uninstall --yes -r $POST_INSTALL_REQUIREMENTS
+# Handle upgrade scenario, remove required packages only when uninstall
+if [ $1 == 0 ]
```
Comment (saurabhkhanvilkar) [inline py-utils/utils-pre-uninstall:7]: what does $1 == 0 mean?...

Comment (sachinpunadikar) [reply py-utils/utils-pre-uninstall:7]: 0 means uninstall .. for upgrade it is 1

Comment (saurabhkhanvilkar) [reply py-utils/utils-pre-uninstall:7]: please consider documenting these options at the start of this script (comments). maybe this is obvious and no need to document. in that case, please ignore my comment.

**Review by saurabhkhanvilkar — APPROVED** [review]

File: py-utils/utils-pre-uninstall (hunk 1 above)
Comment (sachinpunadikar) [reply py-utils/utils-pre-uninstall:10]: Done.

**Review by s-arya — APPROVED** [review]

**Pull request merged**